assets/templates/census-first-results.tmpl

```diff
@@ -0,0 +1,119 @@
+                            {{ localise "ReleasedCensusLink2" .Language 1 }}
+                        </a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+        <div class="ons-grid__col ons-col-6@m ons-u-mb-l">
+            <div class="ons-card u-mt-l u-mb-l">
+                <h2 class="ons-u-fs-m" id="historical-data">{{ localise "HistoricCensusData" .Language 1 }}</h2>
+                <p id="historical-data-desc">{{ localise "HistoricCensusDataSubhead" .Language 1 }}</p>
+                <ul class="ons-list ons-list--dashed">
+                    <li class="ons-list__item">
+                        <a href="/census/historiccensusdata" class="ons-list__link">{{ localise "HistoricCensusDataLink1" .Language 1 }}</a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+        <div class="ons-grid__col ons-col-6@m ons-u-mb-l">
+            <div class="ons-card">
+                <h2 class="ons-u-fs-m" id="census-planning">{{ localise "CensusPlanningAndPreparation" .Language 1 }}</h2>
+                <p id="census-planning-desc">{{ localise "CensusPlanningAndPreparationSubhead" .Language 1 }}</p>
+                <ul class="ons-list ons-list--dashed">
+                    <li class="ons-list__item">
+                        <a href="census/planningforcensus2021" class="ons-list__link">{{ localise "CensusPlanningAndPreparationLink1" .Language 1 }}</a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+        <div class="ons-grid__col ons-col-6@m">
+            <div class="ons-card">
+                <h2 class="ons-u-fs-m" id="contact-us">{{ localise "CensusContactUs" .Language 1 }}</h2>
+                <p id="contact-us-desc">{{ localise "CensusContactUsSubhead" .Language 1 }}</p>
+                <ul class="ons-list ons-list--dashed">
+                    <li class="ons-list__item">
+                        <a href="/census/censuscustomerservices" class="ons-list__link">{{ localise "CensusContactUsLink1" .Language 1 }}</a>
+                    </li>
+                    <li class="ons-list__item">
+                        <a href="/census/2011census/2011censusdata/2011censusadhoctables" class="ons-list__link">{{ localise "CensusContactUsLink2" .Language 1 }}</a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+        <div class="ons-grid__col ons-col-6@m ons-u-mb-l">
+            <div class="ons-card">
+                <h2 class="ons-u-fs-m" id="other-census">{{ localise "OtherCensus" .Language 1 }}</h2>
+                <p id="other-census-desc">{{ localise "OtherCensusSubhead" .Language 1 }}</p>
+                <ul class="ons-list ons-list--dashed">
+                    <li class="ons-list__item">
+                        <a href="https://www.scotlandscensus.gov.uk/" class="ons-list__link ons-external-link">
+                            {{ localise "OtherCensusLink1" .Language 1 }}
```

Comment:
Not sure how sensitive to whitespace the contents of the anchor might be but you could use a `{{-` here